README.pioneer.md

```diff
@@ -0,0 +1,70 @@
+# About Pioneer Schemas
+
+**Note:** If you are writing a Pioneer study using [`shield-studies-addon-utils`](https://github.com/mozilla/shield-studies-addon-utils/pull/263), you do not need to do anything special and data sent using `browser.study.sendTelemetry(payload)` should "just work", as long as the payload adheres to the [`shield-study-addon`](https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/dev/templates/include/telemetry/shieldStudyAddonPayload.3.schema.json) schema.
```

Comment:
```suggestion
**Note:** If you are writing a Pioneer study using [`shield-studies-addon-utils`](https://github.com/mozilla/shield-studies-addon-utils/pull/263), you do not need to do anything special and data sent using `browser.study.sendTelemetry(payload)` should "just work", as long as the payload adheres to the `data.attributes` property in the [`shield-study-addon`](https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/dev/templates/include/telemetry/shieldStudyAddonPayload.3.schema.json) schema.
```

Reply:
If this is too long, we can move the longer version of this note to [`shield-studies-addon-utils`](https://github.com/mozilla/shield-studies-addon-utils) and merely link to those docs here.

Reply:
More detail is good. Thanks!